Core/src/test/java/org/tribuo/util/MeanVarianceAccumulatorTest.java

```diff
@@ -74,4 +89,36 @@ public void testRandomized() {
         Assertions.assertEquals(meanVarPair.getB(),accumulator.getVariance(), DELTA);
     }
 
+    @Test
+    public void testMerge() {
+        SplittableRandom rng = new SplittableRandom(Trainer.DEFAULT_SEED);
+
+        double[] values = new double[4096];
+
+        for (int i = 0; i < values.length; i++) {
+            values[i] = rng.nextDouble();
+        }
+
+        double[] firstValues = new double[2048];
+        double[] secondValues = new double[2048];
```

Comment:
These two accumulators are the same size which is why we didn't catch the bug in the mean calculation. This test should be revised.